webrtc.html

```diff
@@ -8606,6 +8630,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <ol>
                 <li>let <var>sender</var> be the
                 <code><a>RTCRtpSender</a></code> used to send DTMF.</li>
+                <li>Let <var>dtmf</var> be the <code><a>RTCDTMFSender</a></code>
+                associated with <var>sender</var>.</li>
+                <li>If <var>dtmf</var>'s <a>[[\CanInsertDTMF]]</a> internal slot
+                is <code>false</code>, <a>throw</a> an <code>InvalidStateError</code>.</li>
```

Comment:
Should say "throw a newly created" as used elsewhere, which links to the appropriate definition.

Reply:
Looks like we only use "newly created" for rejection of a promise.